src/elf/resolver.rs

```diff
@@ -49,21 +48,21 @@ impl ElfResolver {
 }
 
 impl SymResolver for ElfResolver {
+    // TODO: Need to better handle errors.
     fn find_syms(&self, addr: Addr) -> Vec<(&str, Addr)> {
         let parser = self.get_parser();
+        if let Ok(Some((name, start_addr))) = parser.find_sym(addr, STT_FUNC) {
+            // We found the address in ELF.
+            // TODO: Long term we probably want a different heuristic here, as
+            //       there can be valid differences between the two formats
+            //       (e.g., DWARF could contain more symbols).
+            return vec![(name, start_addr)]
+        }
```

Comment:
@anakryiko I understand this logic is different to what we discussed in person, but let's go with it for the time being and to mirror what we do in `find_addr` below. Will adjust this soon.